.pipelines/samples/README.md

```diff
@@ -28,3 +38,38 @@ It's advised to use different Service Principals to access each environment. Eac
 #### Environment names
 
 The terms Canary and Prod are embedded throughout the sample files. Where your Reference Implementation Prefix is different, these replacements will need to be made.
+
+## Subscription-Vend
+
+The Subscription-Vend pipeline demonstrates governed Subscription Creation. It leverages the AzOps deployment capability, through raising a Pull Request on the main branch with a new Arm template dropped at the correct folder scope.
```

Comment:
Thanks for the work on adding examples for the canary/prod hierarchies. I am a bit hesitant though to elevate subscription creation to be its unique resource, and unique pipeline as it deviate from the generic approach (everything is a resource, and deployment must be invoked at a scope). Could we hold this for and reconsider based on demand and learnings?

Reply:
Totally 👍 
Happy to leave this one here for the time being. Simplifying the subscription creation process (albeit leveraging AzOps) isn't a pattern to be encouraged for other Azure resources.

Aware that there's already several other independent examples of coded/pipeline examples of subscription creation, having one that leverages the AzOps push is going to be preferable where separation is required... :)